srcpkgs/lgi-LuaJIT/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'lgi-LuaJIT'
+pkgname=lgi-LuaJIT
```

Comment:
I think this package actually doesn't have anything to do with `LuaJIT`. Both `LuaJIT` and `lua51` can use what ever contained in that package. So I suggest changing the name to `lua51-lgi`.